Source/SIMPLib/Common/ShapeType.h

```diff
@@ -87,7 +87,7 @@ class SIMPLib_EXPORT ShapeType
    * @param types
    * @return
    */
-  static Types FromQVector(QVector<EnumType> &types);
+  static Types FromQVector(QVector<EnumType>& vec);
```

Comment:
types has more meaning than vec.  It's minor, but this probably shouldn't be changed.